boxes/bootstrap.sh

```diff
@@ -33,10 +33,18 @@ function test {
 
 function test_cmds {
   for browser in chromium webkit firefox; do
-    for box in vanilla react vite; do
+    for box in react vite; do
       echo "$hash:ONLY_TERM_PARENT=1 BOX=$box BROWSER=$browser run_compose_test $box-$browser box boxes"
     done
   done
+
+  # Only run vanilla tests with `chromium` for now, because:
+  # 1. This test includes ClientIVC proof generation in playwright and can take a long time to finish.
+  # 2. Proving in Playwright is flaky for Firefox and Webkit.
+  # 3. The vanilla app expects the contracts to be deployed and address configured in the app during build,
+  # so we would need a separate app build to run tests in parallel.
+  # This can be refactored once other boxes are updated to use browser proving.
+  echo "$hash:ONLY_TERM_PARENT=1 BOX=vanilla BROWSER=chromium run_compose_test vanilla-chromium box boxes"
```

Comment:
If I understood correctly, the test only performs a single tx and they're parallelized...I'd try testing the 3 browsers since we've had firefox-only breakages in the past

Reply:
I think it printed the warning of "This test exceeded the allowed 5 minutes" when I tried
Will give another try (maybe we can do three browsers just for one box even if takes time)

Reply:
Another option could be to make the sandbox container have a script that also deploys (without proving) so browsers only test actual functionality. WDYT?